docs/data/material/components/buttons/buttons.md

```diff
@@ -116,6 +116,10 @@ Use `color` prop to apply theme color palette to component.
 
 {{"demo": "IconButtonColors.js"}}
 
+> Note: You can only use 8 colors with color attribute `inherit | default | primary | secondary | error | info | success | warning`, which gives limited usability of colors. To bring more customization to color we can use `style` attribute.
```

Comment:
```suggestion
:::info
> Note: You can only use 8 colors with color attribute `inherit | default | primary | secondary | error | info | success | warning`, which gives limited usability of colors. To bring more customization to color we can use `style` attribute.
:::
```

Using our Markdown callout notation instead of just a plain quote!
We'll probably need a copywriting fine-tuning here, so asking for @samuelsycamore's review.

Reply:
@danilo-leal do i need to make these changes at my end and open the PR again??

Reply:
No need to open another PR! You'd just commit changes with alterations given in the PR review.